pkg/discovery/dtofactory/node_entity_dto_builder.go

```diff
@@ -228,6 +229,20 @@ func (builder *nodeEntityDTOBuilder) getNodeCommoditiesSold(node *api.Node, clus
 		commoditiesSold = append(commoditiesSold, accessComm)
 	}
 
+	// Label commodities
+	for key, value := range node.ObjectMeta.Labels {
+		label := key + "=" + value
+		labelComm, err := sdkbuilder.NewCommodityDTOBuilder(proto.CommodityDTO_LABEL).
+			Key(label).
+			Capacity(labelCommodityDefaultCapacity).
+			Create()
+		if err != nil {
+			return nil, isAvailableForPlacement, err
+		}
+		glog.V(5).Infof("Adding label commodity for Node %s with key : %s", node.Name, label)
+		commoditiesSold = append(commoditiesSold, labelComm)
+	}
```

Comment:
We need to remove the logic to create `VMPM_ACCESS` commodity for labels above

Reply:
Removed.